helper/schema/grpc_provider.go

```diff
@@ -21,7 +22,15 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 )
 
-const newExtraKey = "_new_extra_shim"
+const (
+	newExtraKey        = "_new_extra_shim"
+	tflogSubsystemName = "sdkv2"
+)
+
+func withLogger(ctx context.Context) context.Context {
+	return tfsdklog.NewSubsystem(ctx, tflogSubsystemName,
+		tfsdklog.WithLevelFromEnv("TF_LOG_SDK_V2"))
```

Comment:
Similarly here, although this has the worse consequence of requiring developer and practitioner changes when the next version is released so I'm a little more hesitant with this one. I'm not sure if I have a better naming choice though, since "SDK" is the name of this project. Maybe terraform-plugin-go should've used the sdk.go subsystem name and TF_LOG_SDK_GO while here this could use sdk.sdk subsystem name and TF_LOG_SDK_SDK (likely to be just generally documented as TF_LOG_SDK). For what its worth, terraform-plugin-framework doesn't use a subsystem, so that could also be an option for now.

Reply:
We definitely could just not use a subsystem, if that's the approach you prefer. Otherwise, I'll pitch `TF_LOG_SDK_HELPER_SCHEMA` as an environment variable? 🤔 

There's an argument to be made, I think, that if we use a subsystem here we should in framework, too. I don't know that I have a compelling argument either way, tbh, except that if we _don't_ use a subsystem, if we want to turn it off but keep the other SDK loggers on, we need to set each of their environment variables. If we put this in a subsystem, we can turn that subsystem off, and leave `TF_LOG_SDK` at whatever we want the other loggers to use.

Reply:
We're going with `TF_LOG_SDK_HELPER_SCHEMA` -- thank you for following through with a good suggestion!